wrong source code in pypi, missing files

Missing files in pypi https://pypi.org/project/capstone/5.0.5/#files **.tar.gz**, not possible to build from source. (probably only for macOS because of using cmake)

Repro:
```console
cd $(mktemp -d)
wget https://files.pythonhosted.org/packages/a4/23/759da7f13c2ce29ffe90ccb45eb61ffcd310b436bfb489f3dbd11fba8776/capstone-5.0.5.tar.gz
tar -xf capstone-5.0.5.tar.gz
cd capstone-5.0.5/src
mkdir build
cd build
cmake -DCMAKE_BUILD_TYPE=Release -DBUILD_SHARED_LIBS=ON -DCAPSTONE_BUILD_TESTS=OFF -DCAPSTONE_BUILD_CSTOOL=OFF -G "Unix Makefiles" ..
```

```js
$ cmake -DCMAKE_BUILD_TYPE=Release -DBUILD_SHARED_LIBS=ON -DCAPSTONE_BUILD_TESTS=OFF -DCAPSTONE_BUILD_CSTOOL=OFF -G "Unix Makefiles" ..
-- PROJECT_VERSION: 5.0.5 CAPSTONE_VERSION: 5.0.5
-- The C compiler identification is AppleClang 16.0.0.16000026
-- The CXX compiler identification is AppleClang 16.0.0.16000026
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Check for working C compiler: /usr/bin/cc - skipped
-- Detecting C compile features
-- Detecting C compile features - done
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Check for working CXX compiler: /usr/bin/c++ - skipped
-- Detecting CXX compile features
-- Detecting CXX compile features - done
Enabling CAPSTONE_ARM_SUPPORT
Enabling CAPSTONE_ARM64_SUPPORT
Enabling CAPSTONE_M68K_SUPPORT
Enabling CAPSTONE_MIPS_SUPPORT
Enabling CAPSTONE_PPC_SUPPORT
Enabling CAPSTONE_SPARC_SUPPORT
Enabling CAPSTONE_SYSZ_SUPPORT
Enabling CAPSTONE_XCORE_SUPPORT
Enabling CAPSTONE_X86_SUPPORT
Enabling CAPSTONE_TMS320C64X_SUPPORT
Enabling CAPSTONE_M680X_SUPPORT
Enabling CAPSTONE_EVM_SUPPORT
Enabling CAPSTONE_MOS65XX_SUPPORT
Enabling CAPSTONE_WASM_SUPPORT
Enabling CAPSTONE_BPF_SUPPORT
Enabling CAPSTONE_RISCV_SUPPORT
Enabling CAPSTONE_SH_SUPPORT
Enabling CAPSTONE_TRICORE_SUPPORT
CMake Error: File /var/folders/7x/qtmkc2790t55sclcdkdgzx780000gn/T/tmp.YenSkupS8b/capstone-5.0.5/src/capstone.pc.in does not exist.
CMake Error at CMakeLists.txt:766 (configure_file):
  configure_file Problem configuring file


CMake Error: File /var/folders/7x/qtmkc2790t55sclcdkdgzx780000gn/T/tmp.YenSkupS8b/capstone-5.0.5/src/capstone-config.cmake.in does not exist.
CMake Error at /nix/store/w00k62fv8r6jvx67avb0yi0grxm641pj-cmake-3.30.5/share/cmake-3.30/Modules/CMakePackageConfigHelpers.cmake:501 (configure_file):
  configure_file Problem configuring file
Call Stack (most recent call first):
  CMakeLists.txt:771 (configure_package_config_file)


CMake Error: File /var/folders/7x/qtmkc2790t55sclcdkdgzx780000gn/T/tmp.YenSkupS8b/capstone-5.0.5/src/cmake_uninstall.cmake.in does not exist.
CMake Error at CMakeLists.txt:811 (configure_file):
  configure_file Problem configuring file


CMake Error at CMakeLists.txt:853 (include):
  include could not find requested file:

    CPackConfig.txt


-- Configuring incomplete, errors occurred!
```

Just ran into this. The issue makes it impossible to `pip install` capstone on Linux too.
```
Building wheels for collected packages: capstone
  Building wheel for capstone (pyproject.toml) ... error
  error: subprocess-exited-with-error
  
  × Building wheel for capstone (pyproject.toml) did not run successfully.
  │ exit code: 1
  ╰─> [62 lines of output]
      /tmp/pip-build-env-57i725b1/overlay/lib/python3.11/site-packages/setuptools/dist.py:330: InformationOnly: Normalizing '6.0.0alpha3' to '6.0.0a3'
        self.metadata.version = self._normalize_version(self.metadata.version)
      INFO:root:running bdist_wheel
      INFO:root:running build
      INFO:root:running build_py
      INFO:__main__:Building C extensions
      INFO:__main__:Build Directory: /tmp/pip-install-u2ij9it1/capstone_38649dc4788a4c04a655df0355129df0/src/build_python
      
      -- The C compiler identification is GNU 12.2.0
      -- The CXX compiler identification is GNU 12.2.0
      -- Detecting C compiler ABI info
      -- Detecting C compiler ABI info - done
      -- Check for working C compiler: /usr/bin/cc - skipped
      -- Detecting C compile features
      -- Detecting C compile features - done
      -- Detecting CXX compiler ABI info
      -- Detecting CXX compiler ABI info - done
      -- Check for working CXX compiler: /usr/bin/c++ - skipped
      -- Detecting CXX compile features
      -- Detecting CXX compile features - done
      -- PROJECT_VERSION: 6.0.0 CAPSTONE_VERSION: 6.0.0
      -- Performing Test SUPPORTS_MU
      -- Performing Test SUPPORTS_MU - Success
      -- Performing Test SUPPORTS_SHADOWING
      -- Performing Test SUPPORTS_SHADOWING - Success
      -- Performing Test SUPPORTS_SUNINIT
      -- Performing Test SUPPORTS_SUNINIT - Failed
      Enabling CAPSTONE_ARM_SUPPORT
      Enabling CAPSTONE_AARCH64_SUPPORT
      Enabling CAPSTONE_M68K_SUPPORT
      Enabling CAPSTONE_MIPS_SUPPORT
      Enabling CAPSTONE_PPC_SUPPORT
      Enabling CAPSTONE_SPARC_SUPPORT
      Enabling CAPSTONE_SYSTEMZ_SUPPORT
      Enabling CAPSTONE_XCORE_SUPPORT
      Enabling CAPSTONE_X86_SUPPORT
      Enabling CAPSTONE_TMS320C64X_SUPPORT
      Enabling CAPSTONE_M680X_SUPPORT
      Enabling CAPSTONE_EVM_SUPPORT
      Enabling CAPSTONE_MOS65XX_SUPPORT
      Enabling CAPSTONE_WASM_SUPPORT
      Enabling CAPSTONE_BPF_SUPPORT
      Enabling CAPSTONE_RISCV_SUPPORT
      Enabling CAPSTONE_SH_SUPPORT
      Enabling CAPSTONE_TRICORE_SUPPORT
      Enabling CAPSTONE_ALPHA_SUPPORT
      Enabling CAPSTONE_HPPA_SUPPORT
      Enabling CAPSTONE_LOONGARCH_SUPPORT
      Enabling CAPSTONE_XTENSA_SUPPORT
      Enabling CAPSTONE_ARC_SUPPORT
      CMake Error at CMakeLists.txt:992 (include):
        include could not find requested file:
      
          CPackConfig.txt
      
      
      -- Configuring incomplete, errors occurred!
      See also "/tmp/pip-install-u2ij9it1/capstone_38649dc4788a4c04a655df0355129df0/src/build_python/CMakeFiles/CMakeOutput.log".
      See also "/tmp/pip-install-u2ij9it1/capstone_38649dc4788a4c04a655df0355129df0/src/build_python/CMakeFiles/CMakeError.log".
      gmake: Makefile: No such file or directory
      gmake: *** No rule to make target 'Makefile'.  Stop.
      error: [Errno 2] No such file or directory: 'libcapstone.so.6'
      [end of output]
  
  note: This error originates from a subprocess, and is likely not a problem with pip.
  ERROR: Failed building wheel for capstone
Failed to build capstone
ERROR: Could not build wheels for capstone, which is required to install pyproject.toml-based projects
```

```
$ uname -a
Linux debian 6.1.0-30-arm64 #1 SMP Debian 6.1.124-1 (2025-01-12) aarch64 GNU/Linux
$ pip --version
pip 23.0.1 from .venv/lib/python3.11/site-packages/pip (python 3.11)
$ python --version
Python 3.11.2
```

--- a/.github/workflows/build-wheels-publish.yml
+++ b/.github/workflows/build-wheels-publish.yml
@@ -255,9 +255,10 @@ jobs:
 
       - name: Build SDist
         run: |
-          cd bindings/python
           python3 -m pip install -U pip build
-          python3 -m build --sdist
+          python3 -m build --sdist bindings/python
+          python3 -m pip install bindings/python/dist/*.tar.gz
+          python3 -c 'import capstone; print(f"Capstone version installed from sdist: {capstone.__version__}")'
 
       - uses: actions/upload-artifact@v4
         with:

--- a/bindings/python/setup.py
+++ b/bindings/python/setup.py
@@ -82,14 +82,17 @@ def copy_sources():
 
     shutil.copytree(os.path.join(BUILD_DIR, "arch"), os.path.join(SRC_DIR, "arch"))
     shutil.copytree(os.path.join(BUILD_DIR, "include"), os.path.join(SRC_DIR, "include"))
+    shutil.copytree(os.path.join(BUILD_DIR, "LICENSES"), os.path.join(SRC_DIR, "LICENSES"))
 
     src.extend(glob.glob(os.path.join(BUILD_DIR, "*.[ch]")))
     src.extend(glob.glob(os.path.join(BUILD_DIR, "*.m[dk]")))
     src.extend(glob.glob(os.path.join(BUILD_DIR, "*.in")))
-    src.extend(glob.glob(os.path.join(BUILD_DIR, "LICENSES/*")))
-    src.extend(glob.glob(os.path.join(BUILD_DIR, "*.TXT")))
+    src.extend(glob.glob(os.path.join(BUILD_DIR, "SPONSORS.TXT")))
+    src.extend(glob.glob(os.path.join(BUILD_DIR, "CREDITS.TXT")))
     src.extend(glob.glob(os.path.join(BUILD_DIR, "ChangeLog")))
     src.extend(glob.glob(os.path.join(BUILD_DIR, "CMakeLists.txt")))
+    src.extend(glob.glob(os.path.join(BUILD_DIR, "CPackConfig.txt")))
+    src.extend(glob.glob(os.path.join(BUILD_DIR, "CPackConfig.cmake")))
 
     for filename in src:
         outpath = os.path.join(SRC_DIR, os.path.basename(filename))

